docs-starlight/src/lib/commands/sidebar/index.ts

```
@@ -0,0 +1,140 @@
+
+        (item as SidebarItem & { items: SidebarItem[] }).items = files.map(file => {
+          const content = fs.readFileSync(path.join(dirPath, file), 'utf-8');
+          const { data } = matter(content);
+          return {
+            label: data.title as string,
+            translations: {},
+            link: data.slug as string,
+            attrs: {}
+          };
+        });
+        delete (item as any).autogenerate;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)